pytest_catchlog.py

```diff
@@ -122,6 +134,15 @@ def __init__(self, config):
         self.formatter = logging.Formatter(
                 get_option_ini(config, 'log_format'),
                 get_option_ini(config, 'log_date_format'))
+        terminal = py.io.TerminalWriter(sys.stderr)  # pylint: disable=no-member
+        self.console = logging.StreamHandler(terminal)
```

Comment:
What do you think about renaming `console` to just `handler`? I would appreciate that, because #25 [makes](https://github.com/abusalimov/pytest-catchlog/commit/4121487982a1b39937f9daa04cfc94c016e201bc#diff-bf8049f1ff38ea40a0da2b88b46ee051R129) use of a single shared handler for both capturing and live streaming. And since we decided to merge your PR first, I'd like to keep the further diff as concise as possible.